action.yaml

```
@@ -20,6 +20,10 @@ inputs:
     description: Github token with read permission to artifacts in given repo.
     required: false
     default: ""
+  directory:
```

```suggestion
  tmp_download_directory:
```

I feel like we should be explicit about the purpose of this variable.

---

yes I thought about it , doc can be updated too

What about download_tmp_directory (to align with later download_output)

---

yeah sure